src/web/trykkomania/README.md

```diff
@@ -28,5 +76,47 @@ Oppgaven er *kun* javascript, men det lønner seg å ha kjennskap til HTML og CS
 [`onclick`]: https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onclick
 [`setInterval`]: https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setInterval
 
-## Løsning
+# Løsning {.challenge}
 [Her er en full løsning av oppgaven.](løsning.js)
+
+
+# Fremgangsmåte
+Her kommer tips, erfaring og utfordringer til de ulike stegene i den faktiske oppgaven. [Klikk her for å se oppgaveteksten.](../trykkomania/trykkomania.html){target=_blank}
```

Comment:
Link: Trenger ikke gå ned en katalog, for deretter gå opp igjen. `[tekst](trykkomania.html)` holder.